android/BOINC/app/src/main/java/edu/berkeley/boinc/client/Monitor.java

```diff
@@ -1003,7 +1013,9 @@ public AccountOut lookupCredentials(AccountIn credentials) throws RemoteExceptio
 		public boolean isStationaryDeviceSuspected() throws RemoteException {
 			try {
 				return Monitor.getDeviceStatus().isStationaryDeviceSuspected();
-			} catch (Exception e) {}
+			} catch (Exception e) {
+				if(Logging.ERROR) Log.e(Logging.TAG,"Monitor.isStationaryDeviceSuspected error: ",e);
```

Comment:
For consistency:  `Monitor.IMonitor.Stub.isStationaryDeviceSuspected`

Reply:
Fixed